### DISCUSSION TITLE
Resolution scale for mobile and display pre-sets for low-end devices that support lazer

### DISCUSSION THREAD

**Discussion by LukaTV939** [ppy/osu#22394] [Ideas] [answered]
I came back to lazer on mobile so I can try the new multitouch feature and the game is still laggy on my Samsung A50. I've averaged like 30-40 fps in main menu and 30 or less while playing. Every time I do anything it just lags so much which is expected. To fix that there should be a option to decrease the resolution scale % and change some settings to make gameplay much smoother, which should also be available on the first time setup.

```
If there's already an issue explaining the same thing I didn't have time to look through every one of them, so I wrote this
```

**Comment by integerrr** [comment, marked as answer]
Performance is a constant work in progress, and mobile platforms are generally not a focus currently, desktop comes first in that regard.

Usually with mobile the recommendation is to use single-threaded mode for better performance - have you tried that yet?

The hope is that eventually performance on mobile should be good without resorting to the use of settings like resolution scaling
